web/src/components/Header/Header.tsx

```diff
@@ -116,8 +118,12 @@ const Header: React.FC<HeaderProps> = ({
     }
   }
 
+  const theme = useSelector(
```

Comment:
can you make a new custom hook (I think there's a hooks folder) and add this to it?

Reply:
yup totally